deployments/anvilbox/environment.py

```diff
@@ -32,7 +32,9 @@ def mksrc(google_project,
           subgraphs,
           flags: int = 0,
           /,
-          prefix: Optional[str] = None):
+          prefix: Optional[str] = None) -> tuple[str, str]:
+    env, project = snapshot.split('_', 1)
+    project, _, _, _ = project.rsplit('_', 3)
```

Comment:
Why was this change made to the dictionary keys? And why is the splitting different here than in the `anvildev` environment file, given that they use the same set of snapshots?

Reply:
Assuming the logic is correct, I think this could be better expressed as 

```suggestion
    project  = '_'.join(snapshot.split('_')[1:-3])
```

Reply:
>Why was this change made to the dictionary keys?

For it to be similar in convention to the HCA dictionary keys.

>And why is the splitting different here than in the `anvildev` environment file, given that they use the same set of snapshots?

It was an oversight.

Your suggestion wasn't implemented verbatim, but a flavor of it was.

Reply:
> For it to be similar in convention to the HCA dictionary keys. Reference https://github.com/DataBiosphere/azul/issues/5081#issuecomment-1485531487.

I can see how this is a good idea, but I don't understand how that comment motivates this change.

Reply:
I brought it up during PL and @hannes-ucsc and I sorted it out.